app/frontend/tests/unit/store/modules/form.getters.spec.js

```diff
@@ -0,0 +1,67 @@
+import { cloneDeep } from 'lodash';
+import { createLocalVue } from '@vue/test-utils';
+import Vuex from 'vuex';
+
+import formStore from '@/store/modules/form';
+
+const localVue = createLocalVue();
+localVue.use(Vuex);
+
+describe('form getters', () => {
+  let store;
+
+  const sampleState = {
+    form: {
+      name: 'ABC'
+    },
+    formList: [
+      {
+        name: 'ABC'
+      }, {
+        name: 'XYZ'
+      }
+    ],
+    formSubmission: {
+      confirmationId: '1234',
+      submission: {
+        data: { field: '123' }
+      }
+    },
+    permissions: ['SUBMIT', 'READ'],
+    submissionList: ['test', 'sub'],
+    version: { type: 'form' }
+  };
+
+  beforeEach(() => {
+    store = new Vuex.Store(cloneDeep(formStore));
+    store.replaceState(cloneDeep(sampleState));
+  });
+
+  it('form should return the state form', () => {
+    expect(store.getters.form).toEqual(sampleState.form);
+  });
+
+  it('formlist should return the state formlist', () => {
+    expect(store.getters.formlist).toEqual(sampleState.formlist);
```

Comment:
sample.form**L**ist

Reply:
Ohhh **that's** why coverage isn't showing on that getter. Thx!